### DISCUSSION TITLE
Cannot run another pocketbase instance

### DISCUSSION THREAD

**Discussion by haiderlikesrust** [pocketbase/pocketbase#823] [Q&A] [answered]
I'm unable to run another pocketbase instance, it uses the db of the old instance which I created for testing and I dont remember email and password it

**Comment by ganigeorgiev** [comment, marked as answer]
You can delete your test `pb_data` directory or specify a new one with the `--dir` flag.

```sh
./pocketbase serve --dir="/path/to/my/new/pb_data"
```
